### PR TITLE
OOP school library

### PR DESCRIPTION
**Pull request by 29td**
### Project milestone: add Person, Student and Teacher classes

_**In this PR the following was implemented:**_

- Created a person class
- Created student class
- Created teacher class

_Cheers Happy Review 😃 _

### CODE REVIEW

File: Teacher.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher
```
Comment (sja-thedude) [inline Teacher.rb]: - [x] Kindly make sure that teacher inherits from person class like
`class Teacher < Person`
- [x] Kindly make sure that parent_permission is added for initialize and super constructors.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @29td, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Professional Readme
✔️ Correct Gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Teacher.rb (hunk 1 above)
Comment (29td) [reply Teacher.rb]: hey @sja-thedude 
_Thank you for review and changes were made 👍_

**Review by rustam-tolipov — APPROVED** [review]
## Hello @29td👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by 29td** [comment]
Hey @sja-thedude 
_Thank you 👍_

**Pull request merged**